lmdeploy/turbomind/deploy/converter.py

```diff
@@ -113,6 +114,50 @@ def copy_tokenizer(model_path: str, tokenizer_path: str,
                     osp.join(triton_models_path, 'tokenizer'))
 
 
+def update_output_format(model_name: str, model_format: str, model_path: str,
+                         output_format: str):
+    """Update output format according to model info."""
+    TORCH_DTYPE_MAP = {torch.bfloat16: 'bf16'}
+    MODEL_NAME_MAP = {'qwen': 'bf16'}
+    model_name = model_name.split('-')[0]
+
+    def _infer_output_format(config):
+        """_infer_output_format."""
+        torch_dtype = getattr(config, 'torch_dtype', None)
+        if torch_dtype:
```

Comment:
What if `torch_dtype` is `fp16`?

Reply:
Default output format is fp16
https://github.com/InternLM/lmdeploy/blob/c8cdf52180060903e50ebe5124ae39a675ae1832/lmdeploy/turbomind/deploy/converter.py#L222